app/views/incomes/estimatedPay.scala.html

```diff
@@ -48,10 +48,9 @@
     employerName = Some(employerName),
     gaCustomTitle = None
 ){
-    <div class="section--branded hmrc flush--bottom soft--top">
-
-        <h2>@Messages("tai.estimatedPay.yourPay")</h2>
-        <p class="h1-heading" id="payBeforeTax">£@MoneyPounds(payBeforeTax.getOrElse(0),0).quantity</p>
+    <div>
+        <h2 class="flush--top">@Messages("tai.estimatedPay.yourPay")</h2>
```

Comment:
We should probably move away from using `Messages` directly and use `messages` which is implicitly passed to the template